keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/SecurityGroupModel.kt

```diff
@@ -26,13 +27,15 @@ data class SecurityGroupModel(
   val region: String,
   val vpcId: String?,
   val inboundRules: Set<SecurityGroupRule> = setOf(),
-  val moniker: Moniker
+  val moniker: Moniker,
+  val tags: Set<Tag> = setOf()
```

Comment:
Tags are required for the Netflix internal implementation to make determinations about the security group.